src/content/en/_shared/contributors/mco.html

```diff
@@ -0,0 +1,24 @@
+<style>
```

Comment:
I was already in _contributors.yaml, forgot those files are auto-build. Ran gulp build and it worked, thx.